_includes/integrations/react-tech-ref.md

```diff
@@ -284,40 +464,177 @@ Use the `tinymceScriptSrc` prop to specify an external version of TinyMCE to laz
 ```
 
 #### `value`
-This property allows the editor to be used as a controlled component by setting the `value` property and using the `onEditorChange` event.
+The HTML content of the editor when operating as a controlled component. 
+
+When this property is different to the current editor content, the editor content
+will be changed to match (within 200 milliseconds) and a undo level will be created.
+When the editor content changes by this mechanism an attempt will be made to
```

Comment:
```suggestion
When the editor content changes by this mechanism, an attempt will be made to
```

Reply:
I put the comma in a different place (just before "however") as I didn't like the flow when putting it here.